src/components/menubar/menubar.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import acmhack from './acmhack.svg';
+import hackLogo from './hack-logo.png';
+import menubtn from './menu.svg';
+import ButtonBar from './buttonbar';
+import { Collapse } from '@material-ui/core';
+
+const StyleHeader = () =>
+	<Helmet>
+		<link href="https://fonts.googleapis.com/css?family=Poppins:400,500" rel="stylesheet"/>
+	</Helmet>;
+
+const styles = theme => ({
+	logohome: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		cursor: 'pointer'
+	},
+	toolbar: {
+		backgroundColor: 'white',
+		color: '#352A3A',
```

Comment:
Is this color derived from somewhere? Right now it's just a magic constant. I'd prefer it if it came from the `theme` object.

Reply:
turns out this coloring is useless lol.